### DISCUSSION TITLE
Generated image hits 100% then goes blank

### DISCUSSION THREAD

**Discussion by dangraves** [lllyasviel/Fooocus#1194] [Q&A] [answered]
I have an old and very underpowered GPU, Nvidia GTX950, its only 2gig. Even with that the images generate, they get to 100% and then just goes away.  I could basically see the completed image and it looked great in the preview. This is the error shown in the command line:

RuntimeError: CUDA error: the launch timed out and was terminated
CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.

**Comment by siriume** [comment, marked as answer]
relaunch with cli argument --fp32-vae

**Reply by dangraves** [reply]
So do I just do this: "run.bat --fp32-vae"

**Reply by dangraves** [reply]
Thank you so much, it worked.  For reference I also added 512x512 image size to the config.txt file, not sure if it was needed but finally got an image to complete!
